src/templates/customization/custom.webpack.config.js

```diff
@@ -0,0 +1,141 @@
+// @ts-check
```

Comment:
Can this file be generated from typescript ? I see js files almost everywhere while it's a typescript project